### DISCUSSION TITLE
How can I trigger a "dark mode" version of a brand?

### DISCUSSION THREAD

**Discussion by coatless** [quarto-dev/quarto-cli#11487] [Q&A] [answered]
### Description

So, I've been toying around more with the `_brand.yml` inside of a Quarto website. Though, I can't see any documentation on enabling a Dark mode. I assumed that this would be possible with the `light` and `dark` keys under `theme`. That said, I did notice that in the pre-release documentation there isn't support for a dark mode logo; however, the docs were scarce with respect to the theme toggle between dark & light. 

If its not possible yet, could this feature be added before v1.6 is released? 

Specifically, maybe some aliases for theme of `brand-dark` & `brand-light` (with `brand` defaulting to light). 

```yml
format: 
   html: 
      theme: 
         light: brand
         dark: brand-dark
```

Dark mode is a fan favorite ;)

**Comment by gordonwoodhull** [comment]
1.6 is just about released. You are correct that dark mode is only partly supported. 

We should be working on dark mode in the 1.7 timeframe.

**Reply by cscheid** [reply]
To give a bit more background -

Besides "we ran out of time", the main reason we didn't implement dark mode for `_brand.yml` in Quarto 1.6 is that dark mode itself could use a revamp, and we didn't want the `_brand.yml` work to be hindered by a weak Quarto feature. We needed to improve a number of SCSS features in 1.6 because of `_brand.yml`; 1.7 will hopefully have the dark mode changes required to make `_brand.yml` dark mode solid.

**Reply by coatless** [reply]
@cscheid like always, no worries. Happy to know the dark mode fan favorite will get better support going forward!

One minor suggestion: Add a note into the pre-release/brand documentation page emphasizing dark mode is not available for branding.

**Comment by coatless** [comment, marked as answer]
Addressed in https://github.com/quarto-dev/quarto-cli/pull/12319 

So, dark mode will be available in a 1.7 prerelease soon (tm)!

**Reply by gordonwoodhull** [reply]
Yeah, we've long had switchable dark mode for themes, now we have "dark brand" which extends this to brands.

Also see #12277, a way to render plots/tables using light and dark brands and switch them using CSS, and #12173 eliminating flashing when loading pages in dark mode. Big upgrade for dark mode in this release!

**Reply by coatless** [reply]
Sorry, I wrote that too quick. I'm 100% aware of dark mode existing in quarto before brand yml; but, now I can use dark mode **with** brand yml. 

(P.S. I saw the other PRs leading into #12319. I held off until I saw the "feature: dark brand" PR get closed (~15 mins 😄 ))
